src/components/search-wizard/QueryLayer.jsx

```diff
@@ -0,0 +1,12 @@
+import { fieldNames } from '../../config';
+
+export default function QueryLayer({ config }) {
+  const id = `query-layer-${config[fieldNames.queryLayers.name]}`;
+
+  return (
+    <div className="input-wrapper input-wrapper--checkbox">
+      <label htmlFor={id}>{config[fieldNames.queryLayers.name]}</label>
+      <input type="checkbox" id={id} className="flex-shrink-0" />
```

Comment:
**polish**: The color of the form input check boxes doesn't look right with the brand blue for deq.

![image](https://user-images.githubusercontent.com/325813/234639247-514199ef-cd77-4e53-854c-31672944a795.png) ![image](https://user-images.githubusercontent.com/325813/234639270-38c3100f-bd4f-483b-bb6d-b61c71db9655.png)

https://tailwindcss.com/docs/hover-focus-and-other-states#checked

The layer selector looks weird also.
![image](https://user-images.githubusercontent.com/325813/234639578-eae77efb-9e34-40df-9506-ca8ddbfe7281.png)

Reply:
I see that you're using the highlight color for radios so many it's not weird b ut they are much larger than the checkboxes which is weird.